src/components/Filters/Filters.tsx

```diff
@@ -51,8 +51,8 @@ const Filters: React.FC<FiltersProps> = ({
   }, [filtersValues, dataPoints]);
 
   const callOnChange = () => {
-    if (typeof onChange === 'function') {
-      onChange(filtersValues);
+    if (typeof onChangeFilter === 'function') {
+      onChangeFilter(filtersValues);
```

Comment:
I don't see any reason to change the name of this property. It's not called only on filters change but on any change within Filters component so `onChange` is completely fine.

Reply:
It created a type conflict because there is onChange prop for SearchBar / input

Reply:
I'm confused now, how prop name in Filters component can clash with a prop in SearchBar component?

Reply:
It is confusing :) Because they are merged in
 `interface ControlModuleProps extends SearchBarProps, FiltersProps { ... }`

Reply:
ah ok, so in that case it would be probably better not extending but recreating them 
```
interface ControlModuleProps {
  onFilterChange: () => {},
  onSearchChange: () => {},
  ...
}
because it's problem of the ControlModule not Filters that there are clashing prop names